src/include/OpenImageIO/strutil.h

```diff
@@ -670,13 +670,15 @@ template<> inline double from_string<double> (string_view s) {
 
 template<> inline int64_t from_string<int64_t>(string_view s) {
     // For conversion of string_view to unsigned int, fall back on strtoll.
-    auto r = strtoll(std::string(s).c_str(), nullptr, 10);
+    std::string s2 = std::string(s);
+    auto r = strtoll(s2.c_str(), nullptr, 10);
```

Comment:
This change is fine, I guess, but I don't understand why it is necessary. What error message did you get with the old code?